src/hooks/posters.ts

```diff
@@ -1,6 +1,7 @@
 import { useState, useMemo, useCallback } from "react";
 import { VenueTemplate } from "types/venues";
 import Fuse from "fuse.js";
+import { sampleSize } from "lodash";
 
 import { DEFAULT_DISPLAYED_POSTER_PREVIEW_COUNT } from "settings";
```

Comment:
This issue isn't from your PR, but may as well fix it while i'm here:

```suggestion
import { useState, useMemo, useCallback } from "react";
import Fuse from "fuse.js";
import { sampleSize } from "lodash";

import { DEFAULT_DISPLAYED_POSTER_PREVIEW_COUNT } from "settings";

import { VenueTemplate } from "types/venues";
```